examples/browser/webpack.config.js

```diff
@@ -64,18 +63,13 @@ module.exports = {
     resolve: {
         extensions: ['.js'],
         alias: {
-            'vs': path.resolve(outputPath, monacoEditorPath),
-            'dtrace-provider': emptyPath,
-            'safe-json-stringify': emptyPath,
-            'mv': emptyPath,
-            'source-map-support': emptyPath
+            'vs': path.resolve(outputPath, monacoEditorPath)
```

Comment:
We don't use bunyan in the frontend and use the webpack to build only the frontend, so these modules are never bundled